requirements/zeromq.txt

```diff
@@ -1,5 +1,5 @@
 -r base.txt
 -r crypto.txt
 
-pyzmq>=2.2.0; python_version < '3.8'
+pyzmq>=17.0.0; python_version < '3.8'
 pyzmq>=19.0.0; python_version >= '3.8'
```

Comment:
Can we change this too since any pip installing into Fedora or Ubuntu 20.04 will fail.  F32 is v18.1, Ubuntu 20.04 I believe is 17.1.2+

Reply:
I believe the purpose in pinning to 19 or higher was that there are wheel files available. It looks like there are wheels for previous versions though.

Reply:
https://pypi.org/project/pyzmq/17.0.0/#files

Reply:
oh, they aren't python3.8 wheels

Reply:
I know, but breaking people just because it makes our testing life easier is asinine - @s0undt3ch and I argued it out till blue in the face and no resolution.  However, I get Fedora 32, and salt will not pip3 install out of the box.

Reply:
looks like 18.1.0 has them though.

Reply:
18.1.1 if you want to include windows.

Reply:
Fedora 31 is 18.0.2, what is Ubuntu 20.04

Reply:
18.1.1 https://pkgs.org/search/?q=python3-zmq

Reply:
ubuntu/debian didn't complain though.

Reply:
Remove both lines and leave just `pyzmq>=17.0.0`.
Locked requirements won't change and everyone stays less blue :grin:

Reply:
k, did that, force pushed the change.

Reply:
Debian / Ubuntu don't appear to check setuptools the way the red hat family do
But someone is going to complain about it